benchmark/sample_genes.py

```diff
@@ -0,0 +1,90 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# Select gene families (orthologous groups) from OrthoFinder results for
+# subsequent analyses
+#
+
+import sys
+import click
+import pandas as pd
+
+
+def sample_genes(ortho_groups_fp,
+                 min_taxa_cutoff=10.0):
+    """ Select gene families (orthologous groups) from OrthoFinder result
+
+    Parameters
+    ----------
+    ortho_groups_fp : str
+        orthologous groups definition file (OrthoFinder output file:
+        Orthogroups.csv)
+    min_taxa_cutoff : float
+        minimum number (if > 1) or fraction (if <= 1) of taxa that contain
+        one or more homologs of the query gene. default: 10.0
+
+    Return
+    ------
+    dict of dict of set of str
+        { ogid : { taxon : set(protein(s)) }
+            ogid: ID of the orthologous group (assigned by OrthoFinder)
+            taxon: ID of the taxon (genome)
+            protein: ID of the protein
+
```

Comment:
you should add a section 'Raises' to document which exceptions can be raised by your function.